UIKitHometasks/UIKitHometasks/ViewController.swift

```diff
@@ -6,14 +6,96 @@
 //
 
 import UIKit
-
+/// controller
 class ViewController: UIViewController {
```

Comment:
```suggestion
class MainViewController: UIViewController {
```
И описание контролер над классом нам ничего не говорит